tests/unit/serialization/custom_global_serialization_test.py

```diff
@@ -42,6 +42,18 @@ def __eq__(self, other):
         return False
 
 
+class TheOtherCustomClass(CustomClass):
+    
+    def __init__(self, uid, name, text, source=None):
+        super(TheOtherCustomClass, self).__init__(uid, name, text, source)
+
+    def __eq__(self, other):
+        super(TheOtherCustomClass, self).__eq__(other)
+
+    def func(self):
+        pass
```

Comment:
do we need this func?

Reply:
No but wanted to add something to show it is different in some way. Gonna delete.